crates/bevy_ecs/src/system/exclusive_function_system.rs

```diff
@@ -216,7 +216,7 @@ pub trait ExclusiveSystemParamFunction<Marker>: Send + Sync + 'static {
 pub struct HasExclusiveSystemInput;
 
 macro_rules! impl_exclusive_system_function {
-    ($($param: ident),*) => {
+    ($(#[$meta:meta])* $($param: ident),*) => {
```

Comment:
I noticed that even though this was added, there is no use of this new var, is it right?